go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rancher/k3s
 
-go 1.13
+go 1.15
```

Comment:
How does this affect downstream a.k.a rke2?

Reply:
There was some discussion about this in https://github.com/rancher/k3s/pull/2098#issuecomment-684163892

Reply:
Just to confirm, I did a local build of rke2 against my branch and it works fine.